includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -63,6 +63,14 @@ class WC_Payment_Gateway_WCPay extends WC_Payment_Gateway_CC {
 	const UPDATE_SAVED_PAYMENT_METHOD     = 'wcpay_update_saved_payment_method';
 	const UPDATE_CUSTOMER_WITH_ORDER_DATA = 'wcpay_update_customer_with_order_data';
 
+	/**
+	 * Set a large limit argument for retrieving user tokens.
+	 *
+	 * @type int
+	 */
+
+	const USER_FORMATTED_TOKENS_LIMIT = 100;
```

Comment:
Not sure about the naming here, what is the intention of `FORMATTED` here?

Reply:
I had initially named it `TOKENS_LIMIT` . Based on suggestion from @dpaun1985 , which I agreed - It was appearing too generic. So drew reference from the name of the [associated function](https://github.com/Automattic/woocommerce-payments/blob/b028841a3dc52564d549d0a1b088e93fdff78734/includes/class-wc-payment-gateway-wcpay.php#L2876)  - `get_user_formatted_tokens`

Let me know if it needs a change, @kalessil .